### PR TITLE
Check Sacura Job status in a Go test

### LINKED ISSUE #350: Sacura Job Success Check in a Go test
## Problem

We check the sacura job success or failure in bash:
https://github.com/knative-sandbox/eventing-kafka-broker/blob/8dcbd27c1ea5820d2aef81457978093c7c2c49b5/test/e2e-tests.sh#L41-L54
That makes it difficult to track when the job fails during continuous testing.

We should wrap that check in a go test that runs in place of the bash-based check.

/area test
/help

### PR DESCRIPTION
**Pull request by pierDipi**
Fixes #350

This PR allows us to get individual test report output for the Sacura test,
so better test infra integration.

## Proposed Changes

- Check Sacura Job status in a Go test

### CODE REVIEW

File: test/e2e/sacura_test.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+	defer testlib.TearDown(c)
+
+	ctx := context.Background()
+
+	const (
+		app       = "sacura"
+		namespace = "sacura"
+	)
+
+	jobPollError := wait.Poll(10*time.Second, 10*time.Minute, func() (done bool, err error) {
```
Comment (slinkydeveloper) [inline test/e2e/sacura_test.go:49]: Make these constants

File: test/e2e/sacura_test.go [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+)
+
+func TestSacuraJob(t *testing.T) {
+
+	c := testlib.Setup(t, false)
+	defer testlib.TearDown(c)
+
+	ctx := context.Background()
+
+	const (
```
Comment (slinkydeveloper) [inline test/e2e/sacura_test.go:44]: Push outside this fn

**Review by slinkydeveloper — COMMENTED** [review]
Sounds interesting

**Comment by pierDipi** [comment]
Ok, it works.

**Comment by slinkydeveloper** [comment]
/lgtm

I wonder, does it makes sense to push it to upstream eventing too?

**Pull request merged**